### DISCUSSION TITLE
Change voice control language

### DISCUSSION THREAD

**Discussion by wilko22** [AndreyPavlenko/Fermata#165] [General]
Samsung galaxy s20, Fermata auto 1.8.11
Where I can change voice control language to English. Now it understand only my local Slovenian language and work very well for search the slovenian words?
Is this maybe in control of youtube settings or...?

W

**Comment by AndreyPavlenko** [comment]
Try changing the interface language in the app settings. I'm not sure, but, perhaps, it could help.
